apps/explorer/src/pages/epochs/useEpochProgress.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMemo } from 'react';
+
+import { useTimeAgo } from '~/utils/timeUtils';
+
+export function useEpochProgress(
+    start?: number,
+    end?: number,
+    suffix: string = 'left'
+) {
+    const [number, label] = useTimeAgo(end).split(' ');
+    const progress = useMemo(
+        () => (start && end ? ((Date.now() - start) / (end - start)) * 100 : 0),
+        [start, end]
+    );
+    return {
+        progress,
+        label: `${number} ${label} ${suffix}`,
```

Comment:
There's no need to split the `useTimeAgo` return, just do `${time} ${suffix}`

Reply:
I originally did this to drop the smaller time denomination. i.e. show "3 hours left" instead of "3 hours 59 seconds left" since I figured it would be an estimate here. 

But I agree - I'll use default behavior and we can adjust if needed